WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift

```diff
@@ -246,12 +250,60 @@ extension SiteTagsViewController {
     }
 }
 
-// MARK: - Navigation
+// MARK: - Navigation to Tag details
 extension SiteTagsViewController {
-    fileprivate func navigate(to tag: PostTag) {
-        let singleTag = SiteTagViewController(blog: blog, tag: tag)
+    fileprivate func navigate(to details: PostTag?) {
+        let data = SettingsTitleSubtitleController.Data(title: details?.name, subtitle: details?.tagDescription)
+        let confirmationStrings = confirmation()
+        let singleTag = SettingsTitleSubtitleController(data: data, confirmation: confirmationStrings)
```

Comment:
It might be a bit more clear if the method signature was to stay `navigate(to tag: PostTag)` and for `singleTag` to be renamed to something referencing the tag details screen.